airship_tempest_plugin/tests/api/shipyard/test_actions.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 AT&T Corp
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+import logging
+
+from airship_tempest_plugin.tests.api.shipyard import base
+
+from tempest import config
+from tempest.lib import decorators
+
+CONF = config.CONF
+LOG = logging.getLogger(__name__)
+
+
+class ActionsTest(base.BaseShipyardTest):
+
+    def _get_action_id(self):
+        resp = self.shipyard_actions_client.list_actions()
+        self.assertTrue(len(resp[1]) > 0,
+                        'No actions available, nothing to test')
+        # get the response body
+        return resp[1]['id']
+
+    def _get_action_step_id(self):
+        resp = self.shipyard_actions_client.list_actions()
+        self.assertTrue(len(resp[1]) > 0,
```

Comment:
not sure, but should this be:
self.assertTrue(len(resp[1]) < 0,
'No actions available, nothing to test')